src/camera/options.ts

```diff
@@ -265,10 +265,18 @@ export const AutoWhiteBalanceSensitivityOption = {
 	},
 }
 
+export function twoDigitHex(n: number): string {
+	return ('0' + n.toString(16)).slice(-2)
+}
+
+export function isValidPreset(n: number): boolean {
```

Comment:
Good call on breaking this out to a separate function and using it in the loop below here.